ccd-definition/CaseEventToFields.json

```diff
@@ -49,8 +49,32 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseEventID": "enterGrounds",
-    "CaseFieldID": "grounds",
+    "CaseFieldID": "orders_option",
     "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageID": 1,
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "groundsForEPOCONTAINS\"59281\""
```

Comment:
```suggestion
    "FieldShowCondition": "groundsForEPO CONTAINS \"59281\""
```
for readability

Reply:
also, could "FALSE" be used if its a forced control or is it reserved?

Reply:
@swalker125 has suggested 'NEVER_SHOW'

Reply:
tomato/tomatoe. Both are better than 59281, NEVER_SHOW makes the most sense to me and will hopefully make it clear it shouldn't ever be shown. DO_NOT_SHOW could also work. KEEP_HIDDEN. Something along those lines 👍